### ISSUE TITLE
FTP Auto Indexing result No new episodes were found.

### ISSUE THREAD

**Issue by daozhao** [PodcastGenerator/PodcastGenerator#431]
**System Information**
Podcast Generator Version: 3.1.1 
Webserver: nginx/1.10.3 (Ubuntu)
PHP Version: 7.0

**Podcast Generator Config**

```
$upload_dir = "media/

```  

**Expected behavior**
i copy the media file(m4a or mp3) in media folder. and use "FTP Auto Indexing" function.

page result: No new episodes were found.

I user https://XXXXX.com/podcast/pg-cron.php?key=XXXXXX . is the same.

nothing media can index.

FTP Auto Indexing

**Actual behavior**
 

**How to reproduce**

**Comment by daozhao** [comment]
This bug may be the same as another bug(#296)

**Comment by flattery103** [comment]
Can you tell the permissions on the uploaded files? does the server have read/write access to the files?
Do you have command line access or only FTP access to your server?

**Comment by daozhao** [comment]
I don't think it is a question of permissions. I have tried setting permissions to 777.

**Comment by coldacid** [comment]
Yeah, so long as the PHP process created by the web server can read the media files and can write into the directory which stores the files, there shouldn't be permissions issues. (Media files/episodes should be 644, directory should be 755.) This probably /is/ #296.

**Comment by soundneedle** [comment]
Should this  https://XXXXX.com/podcast/pg-cron.php?key=XXXXXX **also** create any new XMLs needed for new MP3 files in the directory? I must manually run the "ftp auto indexing feature" first and then the url will add the new MP3s. Is that expected? A two-step process?
